Gulpfile.mjs

```diff
@@ -222,7 +223,8 @@ function getFiles(glob, { include, exclude }) {
 }
 
 function createWorker(useWorker) {
-  const numWorkers = require("os").cpus().length / 2 - 1;
+  // jest-worker defaults
+  const numWorkers = cpus().length - 1;
```

Comment:
If we want to keep the `/ 2` (using all my cores except for one might slow things down, if I'm doing other tasks in parallel), we can use `Math.ceil(require("os").cpus().length / 2 - 1)`. Also, if `cpus().length` is 1 we probably want 1 worker and not 0 :thinking:

Reply:
If CPU length is 1 or 2 (for Intel CPU with hyper-threading), the `numWorkers` is 0 so we don't spawn extra workers and run in the main process instead.